.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -112,53 +116,55 @@ jobs:
 
       # Application pods need to be restarted for the
       # app signals instrumentation to take effect
-      - name: Restart the app pods
-        run: kubectl delete pods --all -n ${{ env.SAMPLE_APP_NAMESPACE }}
+      - name: Restart the app pods and wait for the endpoints to come online
+        id: restart-pod
+        uses: nick-fields/retry@v2
```

Comment:
This action has been problematic for us in other worflows. I'd suggest using built in command line tools for this and wrap the command yourself.

Reply:
Ok I will try that thanks. This PR contains some changes that have already been merged, so I'm opening up another PR here: https://github.com/aws-observability/aws-otel-java-instrumentation/pull/625